### ISSUE TITLE
restore-group: "400: Unable to parse the raw message - invalid"

### ISSUE THREAD

**Issue by Jerther** [GAM-team/got-your-back#349]
Full steps to reproduce the issue:

1. Use google takeout to export emails from a google workspace user
2. Download the takeout archive, and extract it localluy
3. Create a google Groups to receive the emails
4. Install gyb for windows and when prompted for a user, use a super admin
5. gyb --email groupe.archive@mydomain.com --action restore-group --local-folder c:\takeout --use-admin

Expected outcome (what are you trying to do?):
All emails are imported, except those that are too big

Actual outcome (what errors or bad behavior do you see instead?):
A LOT of "400: Unable to parse the raw message - invalid"

Additional info:
There's not much help on this on the net.... All I found was [this SO article](https://stackoverflow.com/questions/25313073/unable-to-find-a-rfc-822-message-that-works-with-google-groups-migration-api) but that doesn't help a mere user ;)

**Comment by Jerther** [comment]
I also found [this thread](https://groups.google.com/g/got-your-back/c/kzi915ny6Fc/m/B8Yn9dhGCwAJ) on your google group. The user reports having problems with Hangouts (Chat) messages, and that could be the problem here also.

**Comment by Jerther** [comment]
Hmm, not sure I follow you on the initial backup. I used Google Takeout with the default settings. Here, chats appear to be selected by default, unless that means nothing when the top checkbock is checked:

![image](https://user-images.githubusercontent.com/5856745/151578996-e7b601ea-7cf9-47cc-8c9c-da2883dc3352.png)

("Clavardage" means "Chat" in french)

**Comment by jay0lee** [comment]
Yes, you'd want to exclude Chat from the Takeout. They can't be migrated back into Gmail or Groups in any meaningful form.

**Comment by Jerther** [comment]
I understand thanks.

Would it be possible for GYB to detect such chat messages and skip them with a proper warning? The message it outputs right now is a bit worrisome and confusing...

Also the documentation does not say much about Chat messages, except in the "--action count" section.
